cEP-0006.md

```diff
@@ -0,0 +1,87 @@
+coala Community Code of Conduct
+===============================
+
+|Metadata|                                   |
+|--------|-----------------------------------|
+|cEP     |0006                               |
+|Version |1.1                                |
+|Title   |coala Community Code of Cpnduct    |
+|Authors |Mariatta Wijaya <mariatta@coala.io>|
+|Status  |Active                             |
+|Type    |Process                            |
+
+coala Community Code of Conduct
+-------------------------------
+
+Like the technical community as a whole, the coala team and community is made
+up of a mixture of professionals and volunteers from all over the world,
+working on every aspect of the mission - including mentorship, teaching, and
+connecting people.
+
+Diversity is one of our huge strengths, but it can also lead to communication
+issues and unhappiness. To that end, we have a few ground rules that we ask
+people to adhere to. This code applies equally to founders, mentors,
+contributors, and those seeking help and guidance.
+
+This isn’t an exhaustive list of things that you can’t do. Rather, take it in
+the spirit in which it’s intended - a guide to make it easier to enrich all
+of us and the technical communities in which we participate.
+
+This code of conduct applies to all spaces managed by the coala project. This
+includes gitter, github, gitlab, the issue tracker, coala events, and any other
+forums created by the project team which the community uses for communication.
+In addition, violations of this code outside these spaces may affect a person's
+ability to participate within them.
+
+If you believe someone is violating the code of conduct, we ask that you
+[report it](community@coala.io).
```

Comment:
hm now I'm a bit afraid of spam, maybe just `community AT coala DOT io` as text or so?